src/MOI_wrapper.jl

```diff
@@ -392,6 +425,17 @@ function MOI.copy_to(cbc_dest::Optimizer, src::MOI.ModelLike;
         CbcCI.setInteger(cbc_dest.inner, column - 1)
     end
 
+    # SOS constraints
+    # type 1
+    for i in eachindex(tmp_model.sos1_weights)
+        CbcCI.addSOS(cbc_dest.inner, 1, Cint[1,length(tmp_model.sos1_weights[i])+1], tmp_model.sos1_indices[i], tmp_model.sos1_weights[i], 1)
```

Comment:
Missing space after comma

Reply:
Or you can do
```Julia
for (indices, weights) in zip(tmp_model.sos1_indices, tmp_model.sos1_weights)
    CbcCI.addSOS(cbc_dest.inner, 1, Cint[1, length(weights) + 1], indices, weights, 1)
end
```

Reply:
Zip allocates some weird stuff, I love using it but it has overhead for nothing